arm-ddwrt-linux-muslgnueabi.cmake

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2017, NeroBurner
+# Copyright (c) 2017, Ruslan Baratov
+# All rights reserved.
+
+if(DEFINED POLLY_ARM_DDWRT_LINUX_MUSLGNUEABI_)
+  return()
+else()
+  set(POLLY_ARM_DDWRT_LINUX_MUSLGNUEABI_ 1)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_init.cmake")
+
+polly_init(
+    "Linux / gcc / DD-WRT / MUSL / c++11 support / cortex-a9"
+    "Unix Makefiles"
+)
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_common.cmake")
+
+string(COMPARE EQUAL "$ENV{ARM_DDWRT_LINUX_MUSLGNUEABI_ROOT}" "" _is_empty)
+if(_is_empty)
+  polly_fatal_error("Environment variable 'ARM_DDWRT_LINUX_MUSLGNUEABI_ROOT' is not set")
+endif()
+
+# set system name, this sets the variable CMAKE_CROSSCOMPILING
+set(CMAKE_SYSTEM_NAME Linux)
+set(CROSS_COMPILE_TOOLCHAIN_PREFIX "arm-ddwrt-linux-muslgnueabi")
```

Comment:
I don't see such prefix in `toolchain.tar.xz` file. You can get the list of available prefixes by searching for GCC compiler:
* arm-linux-gcc
* arm-linux-uclibc-gcc
* arm-openwrt-linux-gcc
* arm-openwrt-linux-muslgnueabi-gcc
* armeb-openwrt-linux-muslgnueabi-gcc
* i486-openwrt-linux-musl-gcc
* mips-openwrt-linux-musl-gcc
* mips64-openwrt-linux-uclibc-gcc
* mipsel-linux-uclibc-gcc
* mipsel-openwrt-linux-musl-gcc
* mipsel-openwrt-linux-uclibc-gcc
* powerpc-linux-uclibc-gcc
* powerpc-openwrt-linux-musl-gcc
* powerpc-openwrt-linux-uclibc-gcc
* x86_64-openwrt-linux-musl-gcc